health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/SetupCampaign.js

```
@@ -70,94 +70,80 @@ function cycleDataRemap(data) {
+    "EQUAL_TO": "==",
+    "NOT_EQUAL_TO": "!=",
+    "IN_BETWEEN": "IN" // Map IN_BETWEEN to a keyword if needed
+  };
+  return operatorMapping[operator] || ""; // Default to empty if not found
+}
+
+function restructureData(data, cycle, DeliveryConfig, projectType) {
+  const tt = DeliveryConfig?.find((e) => e.code === String(projectType));
+
+  if (tt) {
+    delete tt.cycles;   // Remove old cycles
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)